driver/bpf/probe.c

```diff
@@ -10,6 +10,9 @@ or GPL2.txt for full copies of the license.
 
 #include <generated/utsrelease.h>
 #include <uapi/linux/bpf.h>
+#if __has_include(<asm/rwonce.h>)
+	#include <asm/rwonce.h>
+#endif
```

Comment:
I think I'd prefer this to not be indented, I see other `#ifdef` blocks in this same file that keep things sticking to the left. Other than that, if the probe builds correctly, LGTM.

Reply:
Uh you are right!

> Other than that, if the probe builds correctly, LGTM.

Yep, luckily enough i added a driverkit CI job on libs to test the build on 2.6, 3, 4 and 5 major version kernels, therefore we should be safe.
However, i'd keep this for next driver, not for the 4.0.0 release.

Reply:
Makes sense, thanks!